src/main/java/com/mraof/minestuck/tileentity/redstone/StatStorerTileEntity.java

```diff
@@ -82,7 +83,7 @@ public void tick()
 		if(level == null || !level.isAreaLoaded(worldPosition, 1))
 			return;
 		
-		if(tickCycle % 6 == 1) //6 is wireless constant
+		if(tickCycle % WIRELESS_TICK_FREQUENCY == 1)
```

Comment:
I suppose "wireless" is not relevant to this tile entity but it uses the same refresh rate as wireless redstone components for consistency

Reply:
Consistency in what sense?
That in order for them to work correctly together, they need the same refresh rate?
That a player or dungeon builder would expect the same refresh rate across all machine types for their builds?
Or simply that they behave similarly, and therefore may have the same refresh rate?

Reply:
Alright, then it might make sense to declare it as a constant somewhere.
`public static final int PUZZLE_BLOCK_TICK_RATE = 6;`
or
`public static final int STANDARD_TICK_RATE = 6;`
in either `MSTileEntityTypes` or `MinestuckConfig`.
And then you could either refer to this constant directly in the puzzle blocks, or define and use a block-specific constant in each relevant puzzle block.
`private static final int TICK_RATE = MinestuckConfig.PUZZLE_BLOCK_TICK_RATE;`
How does that sound?

Reply:
Sounds good to me, I might put it in the config and allow a minimal level of control over the rate